WordPress/src/main/res/values/strings.xml

```diff
@@ -2352,4 +2352,7 @@
     <string name="site_creation_fetch_suggestions_failed">No connection</string>
     <string name="new_site_creation_verticals_title">TODO</string>
     <string name="new_site_creation_button_skip">Skip</string>
+    <string name="site_creation_error_generic_title">There was a problem</string>
+    <string name="site_creation_error_generic_subtitle">Error communicating with the server, please try again</string>
+    <string name="site_creation_error_connection_title">No internet connection</string>
```

Comment:
Is there a reason we are not using the `no_network_message` resource instead?

Reply:
@SylvesterWilmott  Could we reuse following resource "There is no network available" on the fullscreen error screen?